schemas/list-workertypes-response.yml

```diff
@@ -0,0 +1,31 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "List Worker-Types Response"
+description: |
+  Response from a `listWorkerTypes` request.
+type:               object
+properties:
+  workerTypes:
+    type:           array
+    title:          "WorkerTypes from the Provisioner"
+    description: |
+      List of worker-types in this provisioner.
+    items:
+      type:         string
+      title:        "WorkerType name"
+      description: |
+        WorkerType name.
+      additionalProperties: false
```

Comment:
Also maybe instead of making it a list of strings... let's make it a list of objects... that way we can add additional properties for each workerType in the future (without breaking compatibility).

Ie:
```js
{
 workerTypes: [
    {workerType: '...'},
    ....
  ]
}
```